pkg/log/log.go

```diff
@@ -59,11 +84,19 @@ func DedupedInfof(logTypeLimit int, format string, a ...interface{}) {
 }
 
 func Profilef(format string, a ...interface{}) {
-	klog.V(3).Infof(fmt.Sprintf("[Profiler] %s", format), a...)
+	log.Info().Msgf(fmt.Sprintf("[Profiler] %s", format), a...)
+}
```

Comment:
Yeah, how and what to add is a good question. We could potentially turn the profile logger into a sub logger i.e. https://github.com/rs/zerolog#sub-loggers-let-you-chain-loggers-with-additional-context 
I don't have enough context yet to say where to go with the profile logger, perhaps it should end up behind a flag so it's not always printing logs since profiling is usually resource intensive and not always needed.